dev/integration_tests/android_views/test_driver/main_test.dart

```diff
@@ -22,11 +22,13 @@ Future<void> main() async {
     final SerializableFinder motionEventsListTile =
     find.byValueKey('MotionEventsListTile');
     await driver.tap(motionEventsListTile);
+    await Future<void>.delayed(const Duration(milliseconds: 100));
     await driver.waitFor(find.byValueKey('PlatformView'));
```

Comment:
Do you know why it would hang?
I would have thought this `await driver.waitFor(find.byValueKey('PlatformView'));` should be enough since the driver is going to wait until the `PlatformView` shows up.